client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1938,6 +1938,17 @@ Undocumented.prototype.unregisterDevice = function( deviceId, fn ) {
 };
 
 /**
+ * Requests streamlined approval to WordAds program
+ *
+ * @param {int}       siteId            The site ID
+ * @returns {XMLHttpRequest}          The XHR instance
+ */
+Undocumented.prototype.wordAdsApprove = function( siteId ) {
```

Comment:
Let's try to add this method directly to wpcom.js. I can do it if you are agree.